packages-common/pluggable-widgets-tools/src/typings-generator/generate.ts

```diff
@@ -11,7 +11,9 @@ const mxExports = [
     "ListValue",
     "NativeIcon",
     "NativeImage",
-    "ObjectItem",
```

Comment:
That means that we require Mendix 8.12 or 8.13. Hence, we should bump `pluggable-widgets-tools` version

Reply:
Bumped both, `pluggable-widgets-tools` package version and `mendix` dependency version.